OutOfSchool/OutOfSchool.DataAccess/Repository/Files/GcpFilesStorageBase.cs

```diff
@@ -25,6 +32,12 @@ protected GcpFilesStorageBase(IGcpStorageContext storageContext)
 
         private protected string BucketName { get; }
 
+        public async Task<IAsyncEnumerable<Objects>> GetBulkListsOfObjectsAsync(string prefix = null, ListObjectsOptions options = null)
+        {
+            var objects = await Task.Run(() => StorageClient.ListObjectsAsync(BucketName, prefix: prefix, options: options));
+            return objects.AsRawResponses();
+        }
```

Comment:
Calling `Task.Run` looks redundant here. This method could be simplified.
```suggestion
        public IAsyncEnumerable<Objects> GetBulkListsOfObjects(string prefix = null, ListObjectsOptions options = null)
            => StorageClient.ListObjectsAsync(BucketName, prefix: prefix, options: options).AsRawResponses();
```

Reply:
Yeah, that's indeed true, there are no network requests before the enumerating. Thanks